funsor/distributions.py

```diff
@@ -164,6 +164,14 @@ def eager_delta(v, log_density, value):
     return funsor.delta.Delta(v.name, value, log_density)
 
 
+def LogNormal(loc, scale, value=None):
```

Comment:
We should be able to follow this pattern to create more general transformed distributions.